### PR TITLE
Multiple small fixes

### PR DESCRIPTION
**Pull request by tobloef**
Be careful when merging!

### CODE REVIEW

File: report/11_Systems-Design.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 ### System Design 
-The MiniTwit application is a web app written in C# using ASP.NET. Data is stored using EntityFrameworkCore as a database abstraction layer working with a Postgres database. This stack was chosen since it is a well supported and maintained stack with plenty of available documentation. This means that many third-party packages are available, like SeriLog, which makes interaction with ElasticSearch painless, and Prometheus-net for plug and play communication with Prometheus.
+The new version of the MiniTwit application is a web app written in C# using ASP.NET. Data is stored using Entity Framework Core (EF Core) as a database abstraction layer working with a PostgreSQL database. This stack was chosen because it is both well supported and maintained, as well as well documented. Furthermore, this means that many third-party packages are available, like SeriLog which makes interaction with ElasticSearch painless and Prometheus-net for plug and play communication with Prometheus.
```
Comment (Jakobis) [inline report/11_Systems-Design.md]: I do not think we should call it the new version, since a current version will always be the new version.

Comment (Jakobis) [reply report/11_Systems-Design.md]: Also, the sentence "well supported and maintained, as well as well documented." needs fewer wells. Perhaps just "well supported, maintained, and documented."

**Review by Jakobis — CHANGES_REQUESTED** [review]

File: report/11_Systems-Design.md (hunk 1 above)
Comment (tobloef) [reply report/11_Systems-Design.md]: It was to create a distinction between the original Python version and our ASP.NET version. Do you have any ideas of how to accomplish this in a more clear way?

**Review by Jakobis — APPROVED** [review]

File: report/12_Systems-Architecture.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+The different services are containerized with docker and orchestrated with docker-compose, which also handles startup order. In the current iteration of the system this is all done on a single server as can be seen in figure X (TODO: Update). Each component could be moved to separate servers without major problems, a process that was in progress on the branch [swarm](https://github.com/jlndk/devoops/tree/swarm) but ultimately was not finished in time. More on this in the Scaling section. 
 
 #### Sub systems
 
-Traefik is used as a reverse proxy and Edge Router. As can be seen from the diagram it interacts with MiniTwit.Web.App, Kibana and Grafana. These three are the public facing applications and Traefik is used to manage SSL certificates and HTTPS. Load balancing can be done through Traefik, when using Docker Swarm.
+[Traefik](https://containo.us/traefik/) is used as a reverse proxy and edge router. As shown in figure X (TODO: Update) it interacts with the three public facing applications of the system, MiniTwit.Web.App, Kibana and Grafana, it provides SSL certificates management and HTTPS. Load balancing can also be done through Traefik, when using Docker Swarm.
```
Comment (Hjaltesorgenfrei) [inline report/12_Systems-Architecture.md]: ; after system maybe?

File: report/25_Process-Branching-Strategy.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ## Branching Strategy
 
-Due to the smaller scope and team size, it was decided to use a lightweight, but surprisingly useful, branching strategy called GitHub flow (Github, https://guides.github.com/introduction/flow/, visited: 2020-05-10) (not to be confused with Git flow). GitHub flow works like this:
+Due to the smaller scope and team size, it was decided to use a lightweight, but surprisingly useful, branching strategy called [GitHub flow](https://guides.github.com/introduction/flow/) (not to be confused with Git flow). GitHub flow works like this:
```
Comment (Hjaltesorgenfrei) [inline report/25_Process-Branching-Strategy.md]: Why is it suprisingly useful?

File: report/27_Process-Monitoring.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
+The Digital Ocean dashboards are very basic, especially for uptime moniroting, but they are also incredibly simple to set up and use.
 This was, in the early stages, all the monitoring the system had.
 
-In order to monitor more closely what each process is doing, the project utilizes [Prometheus](https://prometheus.io/docs/introduction/overview/). Prometheus is a systems monitoring toolkit, capable of gathering and storing information about processes running as well as being extended to monitor even more specific things, based on what is running on the system. Standard metrics for monitoring include CPU usage, process memory/CPU/disk usage, the simple low-level information.
+In order to monitor more closely what each process is doing, the project utilizes [Prometheus](https://prometheus.io/docs/introduction/overview/). Prometheus is a systems monitoring toolkit, capable of gathering and storing information about processes running as well as being extended to monitor even more specific things, based on what is running on the system. Standard metrics for monitoring include simple low-level information, such CPU, memory and disk usage for both the system and the specific process.
```
Comment (Hjaltesorgenfrei) [inline report/27_Process-Monitoring.md]: > Standard metrics for monitoring include simple low-level information, such CPU, memory and disk usage for both the system and the specific process.

What is the purpose of this line? This is not standard?

File: report/43_Lessons-Maintaince.md [hunk 5]
```diff
@@ -1,9 +1,8 @@
...
+
 If one were to look at the [pull request (Fix SonarCloud issues #186)](https://github.com/jlndk/devoops/pull/186) it is quite clear that an issue came up when Lighthouse tried to reach the register page, as all the check came back red with a score of 0. 
 
-This issue, as well as similar ones, could probably have been caught in the pull request, if a more rigorous and defined review process was defined. 
-A more defined review process could also have alleviated the problem of not knowing what to look for when going through a pull request, which was a frequent problem over the course of this project.
\ No newline at end of file
+This issue, as well as similar ones, could probably have been caught in the pull request, if a more rigorous and well-defined review process was defined. A tighter review process could also have alleviated the problem of not knowing what to look for when going through a pull request, which was a frequent problem over the course of this project.
```
Comment (Hjaltesorgenfrei) [inline report/43_Lessons-Maintaince.md]: swap tighter for something else. Its weird there. 
Its the review process being more well defined that would have fixed the problem.

**Review by Hjaltesorgenfrei — CHANGES_REQUESTED** [review]

**Review by Hjaltesorgenfrei — APPROVED** [review]

**Pull request merged**